stable/datadog/templates/daemonset.yaml

```diff
@@ -69,16 +73,29 @@ spec:
           - name: DD_APM_ENABLED
             value: {{ .Values.datadog.apmEnabled | quote }}
           {{- end }}
-          - name: KUBERNETES_LEADER_CANDIDATE
+          - name: KUBERNETES_LEADER_CANDIDATE # agent5
+            value: {{ .Values.datadog.leaderElection | quote}}
+          - name: DD_LEADER_ELECTION  # agent6
             value: {{ .Values.datadog.leaderElection | quote}}
           {{- if .Values.datadog.leaderElection }}
-          - name: KUBERNETES_LEADER_LEASE_DURATION
+          - name: KUBERNETES_LEADER_LEASE_DURATION # agent5
             value: {{ default "600" .Values.datadog.leaderLeaseDuration | quote }}
+          - name: DD_LEADER_LEASE_DURATION # agent6
+            value: {{ default "60" .Values.datadog.leaderLeaseDuration | quote }}
+          {{- end }}
+          {{- if .Values.datadog.collectEvents }}
+          - name: DD_COLLECT_KUBERNETES_EVENTS # agent6
+            value: {{.Values.datadog.collectEvents | quote}}
           {{- end }}
-          - name: SD_BACKEND
+          - name: SD_BACKEND  # agent5
             value: docker
           - name: KUBERNETES
             value: "yes"
+          - name: DD_KUBERNETES_KUBELET_HOST
```

Comment:
Can you guard this behind a 1.7+ version check?
See https://github.com/kubernetes/charts/blob/master/stable/datadog/templates/_helpers.tpl#L46 for how to do it

Reply:
This won't work anymore once 1.10 is out. You need to use `semvercompare`. See https://github.com/kubernetes/charts/pull/3369/files#diff-d0b2d71f2c76df54f944e5d7204e9f94R44.